### ISSUE TITLE
[A11Y] ORCID hyperlink has no link text, label or image with a text alternative

### ISSUE THREAD

**Issue by israelcefrin** [pkp/immersion#131]
## Issue Description
The ORCID badge/icon that is presented on the article landing page doesn't have any text or label. Hyperlinks must have an accessible name for their purpose.

## How to reproduce / Current behavior
1. Access a Article landing page of a Journal that uses Immersion theme, e.g.: https://ojs34.demo.publicknowledgeproject.org/index.php/immersion/article/view/919
2. Inspect the ORCID icon beside the author's name
3. See the error

## What application are you using?
 OJS version 3.4

## Stack
- Device: Laptop
- OS: MacOS Sequoia
- Browser: Firefox

## Testing method
- Method:  Manual, Browser Plugin

## Action
Include a text or a meaningful label to the icon. It could be "Author's name ORCID profile page"

## Additional information

<img width="411" height="190" alt="Image" src="https://github.com/user-attachments/assets/4546cf88-e971-4062-a38f-9d147c397028" />

**Comment by israelcefrin** [comment]
Hi @kaitlinnewson , to remediate this issue I will need to add `translate key` to the www/lib/pkp/locale/en/user.po file. Should I file new issue and a PR agains the [pkp-lib repo](https://github.com/pkp/pkp-lib) , or just a PR linking to this issue?

**Comment by kaitlinnewson** [comment]
Hi @israelcefrin, have you checked if this issue exists in the default theme as well? If so, it should be filed in pkp-lib and fixed there, then the themes can all use the locale key. I would suggest targeting 3.5 since that is the branch the translators are working against.
